libsolidity/ast/Types.h

```diff
@@ -1373,6 +1379,8 @@ class MappingType: public Type
 	Type const* keyType() const { return m_keyType; }
 	Type const* valueType() const { return m_valueType; }
 
+	std::vector<Type const*> subtypes() const override { return { m_valueType }; }
```

Comment:
https://github.com/ethereum/solidity/pull/9746#discussion_r484523296
```suggestion
	std::vector<Type const*> subtypes() const override { return {m_valueType}; }
```

Reply:
Done.